WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -576,9 +572,6 @@ import WordPressFlux
         // Rather than repeatedly creating a service to check if the user is logged in, cache it here.
         isLoggedIn = AccountHelper.isDotcomAvailable()
 
-        // Reset our display context to ensure its current.
```

Comment:
Given that we'd be operating directly on the main context now, it's IMHO important not to destroy its cache as navigating back to "My Sites" could be quite slow in that case. I suspect that perhaps this was due to odd `NSManagedObjectContext` cache invalidation issues or some other bug?